### PR TITLE
Remove deprecated commands, organize commands files

### PR DESCRIPTION
**Pull request by ianlowe13**
Phase 2  of "Ian trying to make MCP less full of nonsense or deprecated garbage". Phase 1 was included in the previous ecm-dev merge as that merge contained updates from my earlier modifications.

### CODE REVIEW

File: mcp/tx.c [hunk 1]
```diff
@@ -257,7 +257,8 @@ void SetGyroMask(void)
             t_mask[i] += 1;
             if (t_mask[i] > MASK_TIMEOUT) {
                 if (wait[i] == 0) {
-                    CommandData.power.gyro_off_auto[j] = PCYCLE_HOLD_LEN;
+                    // commented out but this doesn't do anything since... 2011?
```
Comment (evanmayer) [inline mcp/tx.c:260]: Should this commented out code remain?

File: blast_config/tx_struct_tng.c [hunk 2]
```diff
@@ -1027,7 +788,87 @@ channel_t channel_list[] =
...
+    { "ra_1_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_1_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_2_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_2_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_3_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_3_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
+    { "ra_4_p",               I2H,              0.0, TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "dec_4_p",              I2DEG,            0.0, TYPE_INT16, RATE_5HZ, U_NONE, 0 },
 
+    // we should comment this better...
```
Comment (evanmayer) [inline blast_config/tx_struct_tng.c:861]: If it's possible to do so now, do it now

File: blast_config/tx_struct_tng.c [hunk 3]
```diff
@@ -1042,8 +883,26 @@ channel_t channel_list[] =
...
+    // MCP core information
+    { "time",                 SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "time_usec",            SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "parts_sched",          SCALE(CONVERT_UNITY), TYPE_UINT32, RATE_100HZ, U_NONE, 0 },
+    { "status_mcc",           SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "upslot_sched",         SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+    { "veto_sensor",          SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
+
+    // Telemetry
+    { "plover",               SCALE(CONVERT_UNITY), TYPE_UINT16, RATE_5HZ, U_NONE, 0 },
```
Comment (evanmayer) [inline blast_config/tx_struct_tng.c:898]: Is this a magic word? Comment to explain.

File: mcp/motors/actuators.c [hunk 4]
```diff
@@ -55,11 +53,10 @@ extern int16_t InCharge;		/* tx.c */
 
 /* Index for each stepper for structures, name, id */
 #define LOCKNUM 4
-#define HWPRNUM 5
 #define SHUTTERNUM 6
 static const char *name[NACT] = {"Actuator #0", "Actuator #1", "Actuator #2",
-				 "Balance Motor", "Lock Motor", HWPR_NAME, "Shutter", "Pot Valve",
-				 "Pump 1 Valve", "Pump 2 Valve"};
+				 "Balance Motor", "Lock Motor", "FREE_1", "Shutter", "FREE_2",
```
Comment (evanmayer) [inline mcp/motors/actuators.c:58]: Comment that FREE_N are placeholders and available for use.

**Review by evanmayer — CHANGES_REQUESTED** [review]
Once a few minor things are addressed, I'll approve.

This is a tough one to review because of the number of changed lines and the structure of the commits. With code removal mixed into massive reordering, it's hard to see what's been removed. If commits could be structured to separate reordering code and removing code, with commit messages to match, that would make it easier to evaluate what is being removed, and whether any reordering will affect the program.

File: mcp/tx.c (hunk 1 above)
Comment (ianlowe13) [reply mcp/tx.c:260]: fair enough, removed

File: blast_config/command_list.c [hunk 5]
```diff
@@ -63,317 +63,117 @@ const char *linklist_names[] = {0};
...
+    {COMMAND(force_el_on), "force enable el motors despite the pin being in", CONFIRM | GR_MOTOR},
+    // Resets
+    {COMMAND(reset_rw), "reset the serial connection to the RW controller", GR_MOTOR},
+    {COMMAND(reset_piv), "reset the serial connection to the pivot controller", GR_MOTOR},
+    {COMMAND(reset_elev), "reset the serial connection to the elev controller", GR_MOTOR},
+    {COMMAND(reset_ethercat), "reset communications with all EtherCat devices", GR_MOTOR},
+    {COMMAND(restore_piv), "restore the serial settings for the pivot controller", GR_MOTOR},
+
+    /* ACTUATORS */
+    {COMMAND(actbus_on), "turn on the Actuators, Lock, and HWPR", GR_POWER | GR_LOCK | GR_ACT | GR_HWPR},
```
[the remark this answers is not in the document]
Comment (ianlowe13) [reply blast_config/command_list.c:130]: Added in next commit

File: blast_config/tx_struct_tng.c (hunk 3 above)
Comment (ianlowe13) [reply blast_config/tx_struct_tng.c:898]: added

File: mcp/motors/actuators.c (hunk 4 above)
Comment (ianlowe13) [reply mcp/motors/actuators.c:58]: Added

File: blast_config/tx_struct_tng.c (hunk 2 above)
Comment (ianlowe13) [reply blast_config/tx_struct_tng.c:861]: Added some comments based on the command_struct.h declaration of these data parameters

**Review by evanmayer — APPROVED** [review]

**Pull request merged**